### ISSUE TITLE
Reduce logging level to warn in case of dead executions

### ISSUE THREAD

**Issue by ivanparker** [kagkarlsson/db-scheduler#79]
Reduce logging level to warn in case of dead(not found) executions please.

**Comment by kagkarlsson** [comment]
Hi! Which log-line are you referring to? This one?

```
LOG.error("Failed to find implementation for task with name '{}' for detected dead execution. Either delete the execution from the databaser, or add an implementation for it.", execution.taskInstance.getTaskName());```

**Comment by kagkarlsson** [comment]
Why is this line causing you trouble?

**Comment by ivanparker** [comment]
Hi,
most problems we get form message:
"Failed to find implementation for task with name '{}'. Either delete the execution from the database, or add an implementation for it"

Issue is: we create tasks dynamically, and sometimes configuration is changed.
This results in high amount of exceptions in Rollbar.

I suppose error is something serious one should react on, but this case does not require any immediate actions.  

As an option maybe you could add a configurable flag to be able to change logging level.

Thanks

**Comment by kagkarlsson** [comment]
You might get a problem when if you have too many executions without any implementations, because they will block the scheduler from fetching any "real" executions. So they should be removed.

Ideally we should have an option to auto-remove them when we know we have that type of dynamic behavior. Issue #61 addresses this problem, but I haven't landed on what criteria should determine when an execution can be safely removed.

Possibly a default task-handler that removes unknown executions if older than X minutes.

**Comment by kagkarlsson** [comment]
I think the real solution here is to address issue #61 and not reduce log-level?

**Comment by ivanparker** [comment]
Thanks

**Comment by kagkarlsson** [comment]
Fixed in 6.3
